CodeEdit/Settings/ExecutionSettingsView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ExecutionSettingsView.swift
+//  CodeEdit
+//
+//  Created by 朱浩宇 on 2022/3/23.
+//
+
+import SwiftUI
+import TerminalEmulator
+import FontPicker
+import Preferences
+
+struct ExecutionSettingsView: View {
```

Comment:
Why `execution` and not `terminal` ?

Reply:
This will change. Terminal part will move to the theme tab.

Reply:
> The alignment in Terminal Settings is not quite right yet, also make sure all options are available and clear to understand (see review comments).
> 
> **Current Main** <img alt="Screen Shot 2022-03-28 at 12 10 34" width="565" src="https://user-images.githubusercontent.com/9460130/160378177-2a788b57-5fa9-4b70-adb4-2e602efe5999.png">
> 
> **This PR** <img alt="Screen Shot 2022-03-28 at 12 07 42" width="932" src="https://user-images.githubusercontent.com/9460130/160378237-f7ebfad5-5b2c-4bc6-bd48-e543c333c367.png">
<img width="538" alt="CleanShot 2022-03-29 at 20 20 09@2x" src="https://user-images.githubusercontent.com/46642225/160609861-6528803c-333c-4db5-9c28-74bb67a8c0fd.png">

Fixed